recipes/geeadd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "geeadd" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4ff2312cf7304c38f5d16daa503c148b898ddb14a7f04f746fe56b61f7d612f6
+
+build:
+  number: 0
+  entry_points:
+    - geeadd=geeadd.geeadd:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
Since this is a noarch package and you don't support py2, please add update this to:
```
python >3.5
```

Reply:
updated python >3.5